NibiruChain/cw-nibiru
# epic(shifter): smart contract for peg shift and depth shift

- refactor(state.rs)!: use cw_serde new whitelist structure
- refactor(msgs.rs)!: use cw_serde + new whitelist structure"
- feat(whitelist): impl contract entry points for AddMember and RemoveMember
- test(whitelist): test ExecuteMsg
- feat(whitelist): admin check fns, query impl, + tests
- (wasm-out.sh): better output message. It describes the failure case
- build(artifacts)
- refactor(bindings-perp): add NibiruExecuteMsg convenience methods as an impl of the struct
- feat(whitelist): add depth shift and peg shift
- feat(whitelist): use wrapped enum to handle responses for Response<NibiruExecuteMsg> and Reponse<Empty>
- feat(whitelist): use wrapped enum to handle responses for Response<NibiruExecuteMsg> and Reponse<Empty>
- feat(shifter): create ChangeAdmin execute msg
- feat(nibiru-bindings): add routes to the external lib
- feat(bindings-perp): create no op handler
- epic(shifter): implement a valid contract with peg shift, depth shift, and a mutable whitelist
- build: new artifacts
